lib/ansible/modules/windows/win_disk_facts.ps1

```diff
@@ -0,0 +1,55 @@
+#!powershell
+# This file is part of Ansible
```

Comment:
I don't think we still add that specific line to files. I noticed it was being removed when updating the copyright/license statement earlier during a rewrite.

Reply:
Maybe you want to add your copyright statement (as you did with the documentation) ?

Reply:
Thanks!
Ok, I have changed the header as you have proposed.

I would like to add some information about the partitions and volumes of the disks to the output.
Is this a good idea or not needed?